src/main/java/frc/robot/RobotContainer.java

```diff
@@ -62,15 +62,23 @@ private void configureButtonBindings() {
         Joystick rightJoystick = new Joystick(1);
         Joystick altJoystick = new Joystick(2);
```

Comment:
This is the nitpickiest thing ever, but Tim already got to all the good stuff. So with that being said, `joystickRight` and `joystickAlt` might be more consistent with the `btn______` naming scheme you all seem to have settled on.